src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -749,6 +749,9 @@ private static String processPageSourceForGodMode(String content) {
         return content
                 .replaceAll("<#comment[ ]*</#comment>", "<!---->")
                 .replace(Config.APP_URL, "${app.url}")
+                .replace("http://localhost:8888", "${web.url}")
+                .replace(Config.APP_URL.replace("http", "https"), "${web.url}")
+                .replace("\"/_ah", "\"${web.url}/_ah")
```

Comment:
Ah this is one of the things that God mode didn't regenerate correctly previously. Good that it's solved!